From: Gianfranco Costamagna <locutusofborg@debian.org>
Date: Thu, 18 May 2017 10:17:24 +0000
Subject: Part of email attachment 'debdiff'

Signed-off-by: Jordan Justen <jordan.l.justen@intel.com>
---
 Makefile.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile.in b/Makefile.in
index 949d013..62a4c33 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -44,6 +44,8 @@ INSTALL		= @INSTALL@
 INSTALL_PROGRAM	= @INSTALL_PROGRAM@
 INSTALL_DATA	= @INSTALL_DATA@
 
+INSTALLROOT	= $(DESTDIR)
+
 NROFF		= @NROFF@
 ASCIIDOC	= @ASCIIDOC@
 XMLTO		= @XMLTO@
